content/en/developers/ide_integrations/idea/_index.md

```diff
@@ -0,0 +1,155 @@
+
+### Link a service
+
+To provide relevant data from the Datadog platform, add relevant services to a project:
+1. With your project open in IDEA, open the **Datadog** tool window and click the plus icon (**+**).
+1. Search for and select the services that you want to add to the current project.
+
+To remove a service, select it in the **Services** table and click the minus icon (**-**).
+
+<div class="alert alert-info">The names of linked services persist with the project when you close it.</div>
+
+## Using the plugin
+
+After you add a service to your project, right-click on the service and click **Open in Profiling** to open a Profiling tab for the service. A profiling tab displays data for only one service, but you can have multiple tabs open simultaneously.
+
+The Profiling tab shows Continuous Profiling information for the service in a selected environment, aggregated over a specific time frame. Available views are:
+- [Top list](#top-list): Displays a list of the most resource intensive methods for the current profiling measure.
+- [Flame graph](#flame-graph): A flame graph representing stack traces in the profiles.
+
+{{< img src="/developers/ide_integrations/idea/profiling-example1.png" alt="An example of the Profiling tab for a service named product-recommendation" style="width:100%;" >}}
+
+You can specify the following parameters for the profiling data:
+- The profile type to be displayed
+- The environment in which the service is running
+- The time frame for the profile samples to be aggregated
+
+The available profiling types usually include options like **CPU Time** and **Allocated Memory**, but are determined by the platform and vary by language.
+
+## Top list
+
+The **Top List** sub-tab shows the methods that consume the most resources based on the aggregated profile data loaded from the Datadog servers. **Top List** is designed to show a summary of the methods that are most likely to be interesting from a resource consumption point of view.
+
+{{< img src="/developers/ide_integrations/idea/top-list.png" alt="The Top-list view" style="width:100%;" >}}
+
+Double-clicking an item in the list, or selecting **Jump to Source** item from the context menu, opens a source code editor showing where the method is defined.
+
+{{< img src="/developers/ide_integrations/idea/jump-to-source.png" alt="Select Jump to Source from the context menu" style="width:100%;" >}}
+
+To see a flame graph visualization of a method, select the **Search in Flame Graph** item from the context menu.
+
+### Call tree
+
+The call tree to the right of the method list shows the paths that lead to (and from) the selected method.
+
+{{< img src="/developers/ide_integrations/idea/call-tree.png" alt="The method call tree" style="width:100%;" >}}
```

Comment:
This image shows the now-removed `Code Sync` button. An updated version is here: https://user-images.githubusercontent.com/1835893/195762411-9042341e-55de-4aec-af31-8e57caef0c13.png